app/assets/javascripts/package.json

```diff
@@ -14,6 +14,7 @@
     "pretty-text",
     "select-kit",
     "truth-helpers",
-    "wizard"
+    "wizard",
+    "rtlcss-miniracer"
```

Comment:
are we sure we want it in our app/assets ? will it be included in our bundle somehow?

Reply:
I don't think it would sneak into app JS bundle for production, but there's no reason to put this in our main `package.json` so I've removed it.